reactioncommerce/reaction
# feat: Update sidebar, fix mobile responsiveness

Impact: **minor**  
Type: **feature**

## Issue

- Sidebar design is out-of-date with current designs
- Sidebar isn't closable on mobile

## Solution

- Update sidebar to current design spec
- Fix mobile responsiveness
- Make it easier to create toolbars without having to re-implement the same logic over and over

## Breaking changes

`uiState.isLeftSidebarOpen` renamed to `uiState.isPrimarySidebarOpen`.

## Testing
1. Run the app, and login into the admin
2. See the new sidebar
3. Click around
4. Resize to mobile
5. Open / close sidebar
6. With the sidebar open, click on a menu item
7. The sidebar should close and navigate you to that page
8. Resize back to desktop
9. Continue to use the sidebar
